docs/RELEASE_POLICY.md

```diff
@@ -86,7 +86,7 @@ also be specifying the supported kernel versions.
 
 | Release | Release Date | Latest Patch | Min. end of support | Official end of Support        |
 |--------:|-------------:|-------------:|--------------------:|:-------------------------------|
-|    v1.5 |   2023-10-09 |       v1.5.0 |          2024-04-09 | Supported                      |
+|    v1.5 |   2023-12-04 |       v1.5.1 |          2024-04-09 | Supported                      |
```

Comment:
We usually dont update the `Release Date` here (because the support window starts when the minor is released, and the patch release doesnt update it)